### DISCUSSION TITLE
Neovim menu doesnt show icons

### DISCUSSION THREAD

**Discussion by KunalMaverick** [jdhao/nvim-config#213] [Q&A]
Im using Nerd fonts and the menu isnt showing icons for some weird reason
<img width="956" alt="Screenshot 2023-07-31 at 9 24 58 AM" src="https://github.com/jdhao/nvim-config/assets/40132560/2d22ff88-c0dc-41df-a84f-e00d56cbcb8d">

below img is my Iterm config
<img width="588" alt="Screenshot 2023-07-31 at 9 25 32 AM" src="https://github.com/jdhao/nvim-config/assets/40132560/bc50ac87-c0ee-456a-a8b4-58df432a262f">

please help 
thank you ^_^

**Comment by KunalMaverick** [comment]
<img width="978" alt="image" src="https://github.com/jdhao/nvim-config/assets/40132560/03c4a72f-0b25-4c31-9ed2-6febf6125366">
I feel this is the issue, How do i solve this??

**Reply by jdhao** [reply]
are you using latest version of nerdfont? Also check this post: https://jdhao.github.io/2023/07/25/wezterm_icon_missing/

**Reply by David-Leatham** [reply]
Hi,
so I have been using neovim for a very short time and am very happy with your Configuration jdhao. I followed the setup steps for windows and everything is working very well. Also for me, the icons are the only problem I am having. I would like to continue using nvim, but having no file icons is really off-putting. That is why I would be very interested to know what I could be missing after following your setup. For me, all icons are differently colored question marks. In the dashboard, the nvim-tree and with :NvimWebDeviconsHiTest. I should have the latest nerdfont, when I simply install the package nvim-tree/nvim-web-devicons, am I right? And the nerdfix article of yours, I dont understand :(

**Reply by jdhao** [reply]
@David-Leatham  you need to install nerdfonts on your System. Also which terminal are you using? You need also set the font your terminal use.

**Reply by David-Leatham** [reply]
Thank you very much for the quick answer. It is working now that I installed and changed to a nerdfont. I am using windows terminal with powershell. Would you recommend something different?

**Reply by jdhao** [reply]
I think windows Terminal is a good option. You can also try Wezterm
